OracleLinuxDevelopers/oraclelinux8/haproxy/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+FROM ghcr.io/oracle/oraclelinux:8
+
+RUN \
+    dnf -y install --setopt=tsflags=nodocs haproxy && \
```

Comment:
We can't use the `nodocs` option because that suppresses the installation of the `LICENSE` file and we have a legal obligation to provide this file.

Reply:
added the `EXPOSE` directives and removed the `nodocs` option.